File: 1037.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = Number(lines.shift());
if (n >= 0 && n <= 25){
    console.log("Intervalo [0,25]")}
if (n > 25 && n <= 50){
    console.log("Intervalo (25,50]")}
if (n > 50 && n <= 75){
    console.log("Intervalo (50,75]")}
if (n > 75 && n <= 100){
    console.log("Intervalo (75,100]")}
if (n < 0 || n > 100){
    console.log("Fora de intervalo")}

File: 1037.cpp
#include <iostream>
using namespace std;

int main(){
    double n;
    cin >> n;
    if(n >= 0 and n <= 25){
        cout << "Intervalo [0,25]" << endl;
    }
    if(n > 25 and n <= 50){
        cout << "Intervalo (25,50]" << endl;
    }
    if(n > 50 and n <= 75){
        cout << "Intervalo (50,75]" << endl;
    }
    if(n > 75 and n <= 100){
        cout << "Intervalo (75,100]" << endl; 
    }
    if(n < 0 or n > 100){
        cout << "Fora de intervalo" << endl;
    }
}
